tox.ini

```diff
@@ -50,33 +50,9 @@ commands =
     check-manifest
     python setup.py check -m -r -s
 
-[lint]
-deps = -r{toxinidir}/tools/requirements/lint.txt
-
 [testenv:lint]
 skip_install = True
-basepython = python3
-deps = {[lint]deps}
-commands_pre =
-commands =
-    flake8
-    isort --check-only --diff
-
-[testenv:lint-py2]
-skip_install = True
-basepython = python2
-deps = {[lint]deps}
-commands_pre =
-# No need to flake8 docs, tools & tasks in py2
-commands =
-    flake8 src tests
-    isort --check-only --diff
-
-[testenv:mypy]
-skip_install = True
-basepython = python3
-deps = -r{toxinidir}/tools/requirements/mypy.txt
 commands_pre =
+deps = pre-commit
 commands =
-    mypy src
-    mypy src -2
+   pre-commit run --all-files --show-diff-on-failure
```

Comment:
I would add `[]` to this so that we can pass a hook id directly, like `tox -e lint -- mypy`